src/Service/Ecr/tests/Unit/Result/GetAuthorizationTokenResponseTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AsyncAws\Ecr\Tests\Unit\Result;
+
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Test\Http\SimpleMockedResponse;
+use AsyncAws\Core\Test\TestCase;
+use AsyncAws\Ecr\Result\GetAuthorizationTokenResponse;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpClient\MockHttpClient;
+
+class GetAuthorizationTokenResponseTest extends TestCase
+{
+    public function testGetAuthorizationTokenResponse(): void
+    {
+        self::fail('Not implemented');
+
+        // see example-1.json from SDK
+        $response = new SimpleMockedResponse('{
+            "authorizationData": [
+                {
+                    "authorizationToken": "QVdTOkN...",
+                    "expiresAt": "1470951892432",
+                    "proxyEndpoint": "https:\\/\\/012345678901.dkr.ecr.us-west-2.amazonaws.com"
```

Comment:
AWS does not escape the JSON in the sample response here https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_GetAuthorizationToken.html

Does AsyncAws failed when you don't escape it?

Reply:
this was done by `./generate`!

Reply:
removed the escaping